app/src/main/kotlin/com/mapzen/erasermap/view/InstructionGroup.kt

```diff
@@ -70,12 +71,16 @@ class InstructionGroup(val travelType: TravelType, val travelMode: TravelMode,
     return numberOfStops
   }
 
-  fun transitInstructionSpannable(instruction: Instruction): SpannableString? {
+  fun transitInstructionSpannable(context: Context, instruction: Instruction): SpannableString? {
     if (transitInstruction == null) {
       var turnInstruction = instruction.getHumanTurnInstruction()
       val pattern = Pattern.compile("\\([0-9]+ stops\\)")
       val matcher = pattern.matcher(turnInstruction)
       turnInstruction = matcher.replaceAll("")
+      val transitStops = instruction.getTransitInfo()?.getTransitStops() as ArrayList<TransitStop>
+      val departureTime = transitStops[0].getDepartureDateTime().substring(11, 16)
```

Comment:
Created a ticket here to convert these `TransitStop` properties to `Date` objects which will make interaction with this property more safe